internal/core/command/init.go

```diff
@@ -189,7 +189,7 @@ func listenForConfigChanges() {
 	registryClient.WatchForChanges(registryUpdates, registryErrors, &WritableInfo{}, internal.WritableKey)
 
 	signals := make(chan os.Signal)
-	signal.Notify(signals, syscall.SIGINT, syscall.SIGTERM)
+	signal.Notify(signals, syscall.SIGINT, syscall.SIGTERM, os.Interrupt)
```

Comment:
In this case and others like it, just replace `syscall.SIGINT` with `os.Interrupt`. If you look at the definitions in the std lib, the latter points to the former.